### ISSUE TITLE
Unable to open a diagram exported from the web modeler

### ISSUE THREAD

**Issue by ingorichtsmeier** [camunda/camunda-modeler#3221]
### Describe the bug

After modelin a bpmn diagram in the web modeler I donwloaded it to my local computer and would like to open it in the desktop modeler.

The modeler responded with:
![grafik](https://user-images.githubusercontent.com/7870843/194505507-43272fdd-4ace-4cd5-b006-3539fd245337.png)

### Steps to reproduce

1. Model this diagram in the web modeler (A private URL for a propects orgianzation: https://modeler.cloud.camunda.io/diagrams/038ab2aa-7ff2-4ff7-ad58-6ab3073ae5d4--mre4jl-unterprozess?v=632,355,1)
2. Download it to the local computer
3. Open in Camunda Modeler
4. Model could not be opened

This is the downloaded BPMN file: 
[mre4jl-unterprozess.bpmn.txt](https://github.com/camunda/camunda-modeler/files/9731992/mre4jl-unterprozess.bpmn.txt)

### Expected behavior

A seamless modeling experience, open the model without any issue.  

### Environment

Camunda Modeler system information
 * Version: 5.3.0
 * Operating System: Windows 10 amd64
 * Plugins: Cloud Connect, bpmn-js Token Simulation, Migration, Transaction Boundaries, Camunda Property Infos, Tooltip Plugin
 * Execution Platform: BPMN - Camunda Platform 8

**Comment by nikku** [comment]
The broken part is this:

```xml
          <zeebe:input source="&#34; target="InputVariable_0k78fn4" />
```

Are you still able to edit the file in web modeler? The BPMN 2.0 is just structurally broken.

**Comment by nikku** [comment]
You played around in the XML editor? If so, please close this issue.

**Comment by ingorichtsmeier** [comment]
No, We played around in the property panel and added an incomplete input mapping. We didn't touch the XML.

I tried to reproduce by entering a single doublequote, but it was formatted fine when I downloaded again.

Maybe it's a timing issue with a busy web browser running with many open tabs and a lot of javascript? 

I will close this issue. Thank you for your help!

**Comment by nikku** [comment]
Generally speaking we should _never_ generate such broken XML. So if you come across it again, please try to reproduce it / report it.

Broken BPMN 2.0 after editing in our tools is something we should fix.

**Comment by ingorichtsmeier** [comment]
I ran into the situtation again. Here is a screenshot from the web modeler:

![grafik](https://user-images.githubusercontent.com/7870843/195114419-2bccdc9a-aba8-46e8-9a0c-3f99ad1a2bdb.png)

the XML exported contains:
```
    <bpmn:serviceTask id="Activity_0n60yt6" name="Setze BKS = STS">
      <bpmn:extensionElements>
        <zeebe:taskDefinition type="noop" />
        <zeebe:ioMapping>
          <zeebe:output source="&#34; target="BKS = STS" />
        </zeebe:ioMapping>
      </bpmn:extensionElements>
      <bpmn:incoming>Flow_12qqpr1</bpmn:incoming>
      <bpmn:outgoing>Flow_0vvt0fj</bpmn:outgoing>
    </bpmn:serviceTask>
```

**Comment by nikku** [comment]
I'll fuzzy test this tomorrow and see if I can reproduce it. Thanks for reporting.

**Comment by pihme** [comment]
Related Web Modeler issue: https://github.com/camunda/web-modeler/issues/2532
